hipparchus-core/src/main/java/org/hipparchus/analysis/differentiation/DerivativeStructure.java

```diff
@@ -1203,4 +1204,15 @@ public int hashCode() {
         return 227 + 229 * getFreeParameters() + 233 * getOrder() + 239 * MathUtils.hash(data);
     }
 
+    /** {@inheritDoc}. */
+    @Override
+    public boolean hasNullDerivatives() {
+
+        double sumDerivatives = 0.;
+
+        for (int i = 1; i < data.length; i++) {
+            sumDerivatives += FastMath.abs(data[i]);
+        }
+        return sumDerivatives < Precision.EPSILON;
```

Comment:
Hi @wardev,

Thanks for the quick review!  
I've updated my PR with a tolerance and a better Javadoc.  
Is that ok for you? Would you like me to change the method name?

Reply:
Looks good to me!

Reply:
@wardev I'm closing this PR since @Serrof told me that there is already a method to achieve the required goal (see [this thread on Orekit forum](https://gitlab.orekit.org/orekit/orekit/-/issues/1151#note_9809)).

The idea is to do `this.subtract(this.getReal()).isZero();` to check if the Field is Zero, and it's much more generic than what I was trying to do.